src/clj/rems/application/events.clj

```diff
@@ -29,7 +29,8 @@
 (s/defschema CommentedEvent
   (assoc EventBase
          :event/type (s/enum :application.event/commented)
-         :application/request-id s/Uuid
+         :application/request-id (s/maybe s/Uuid)
+         :application/public s/Bool
```

Comment:
hmm optional in command but not in event. I would've assumed the opposite. Now event compatibility is broken for no reason.